IPython/terminal/ipapp.py

```diff
@@ -154,6 +154,7 @@ def make_report(self,traceback):
 
 aliases = dict(base_aliases)
 aliases.update(shell_aliases)
+aliases['term-title-format'] = 'TerminalInteractiveShell.term_title_format'
```

Comment:
D'oh, I missed-grep'ed:

```
 ~/dev/ipython origin/pr/10541 $ ipython --help-all | grep -e 'term_title_format' -A3
--TerminalInteractiveShell.term_title_format=<Unicode>
    Default: 'IPython: {cwd}'
    Customize the terminal title format.  This is a python format string.
    Available substitutions are: {cwd}.

~/dev/ipython origin/pr/10541 $ git diff
diff --git a/IPython/terminal/ipapp.py b/IPython/terminal/ipapp.py
index 11c1503a4..19f419999 100755
--- a/IPython/terminal/ipapp.py
+++ b/IPython/terminal/ipapp.py
@@ -154,7 +154,6 @@ def make_report(self,traceback):

 aliases = dict(base_aliases)
 aliases.update(shell_aliases)
-aliases['term-title-format'] = 'TerminalInteractiveShell.term_title_format'

 #-----------------------------------------------------------------------------
 # Main classes and functions
```

You probably did not knew about the --help-all flag. 
It's missing from the configuration file as the configuration file only get the commented options IPython knew about at config file creation. You you would need to nuke your file , and re-do an `ipython profile create`